src/binary_agreement/mod.rs

```diff
@@ -66,7 +66,7 @@
 mod binary_agreement;
 mod bool_multimap;
 pub mod bool_set;
-mod sbv_broadcast;
+pub mod sbv_broadcast;
```

Comment:
Yes, I don't think that algorithm on its own is generally useful and maybe it would just be confusing to expose it. Maybe we should just re-export `sbv_broadcast::Message as SbvMessage` from `binary_agreement`?